specs-go/config.go

```diff
@@ -112,12 +112,11 @@ type Platform struct {
 
 // Mount specifies a mount for a container.
 type Mount struct {
-	// Destination is the path where the mount will be placed relative to the container's root.  The path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
+	// Destination is the absolute path where the mount will be placed in the container.
 	Destination string `json:"destination"`
 	// Type specifies the mount kind.
-	Type string `json:"type,omitempty"`
-	// Source specifies the source path of the mount.  In the case of bind mounts on
-	// Linux based systems this would be the file on the host.
+	Type string `json:"type,omitempty" platform:"linux,solaris"`
```

Comment:
OK, that's done. But I've been consistent with the existing `User` markdown. Specifically, it does not have a training `-` after Linux and Solaris as per your example. No use of the word `additional`. And uses the term `fields` rather than `properties`. 

Update pushed.